frontend/src/main/java/com/google/tripmeout/frontend/error/WrongFormatUriException.java

```diff
@@ -0,0 +1,20 @@
+package com.google.tripmeout.frontend.error;
+
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import java.lang.Throwable;
+import javax.servlet.http.HttpServletResponse;
+
+public class WrongFormatUriException extends TripMeOutException {
```

Comment:
Nit: I might rename this to `BadUriException` or something otherwise more general. The URI could be bad for a variety of reasons (not just because of its syntax per se).